Add links to AHB eligible RHEL and SLES offers (#96)
MicrosoftDocs/azure-compute-docs: articles/virtual-machines/linux/azure-hybrid-benefit-linux.md

ms.custom: kr2b-contr-experiment, linux-related-content, devx-track-azurecli
---

# Azure Hybrid Benefit for Red Hat Enterprise Linux (RHEL) and SUSE Linux Enterprise Server (SLES) Virtual Machines

Azure Hybrid Benefit (AHB) for Linux lets you easily switch the software subscription model for your VM, enabling flexibility and savings.
The conversion between subscription models is accomplished without a redeployment of your VM, avoiding any downtime risk.

> There are two subscription models on Azure which you can switch seemlessly between using Azure Hybrid Benefit. 

Reviewer: ```suggestion
There are two subscription models on Azure which you can switch seamlessly between using Azure Hybrid Benefit. 
```

* The first enables you to bring your own REHL or SLES subscription directly to Azure.
With this Bring-Your-Own-Subscription (BYOS) model, you only pay Azure for the infrastructure costs of your VM as the software fee is covered by your RHEL or SLES subscription.

> * The second Pay-As-You-Go (PAYG) model enables you to pay for RHEL and SLES subscriptions as you use them. 

Reviewer: ```suggestion
* The second pay-as-you-go model enables you to pay for RHEL and SLES subscriptions as you use them. 
```

This article further defines BYOS and PAYG subscription models, compares the benefits of each model, and shows how you can use the Azure Hybrid Benefit to switch between the two at any point. 
This process applies to Virtual Machine Scale Sets, Spot Virtual Machines, and custom images. 
Azure Hybrid Benefit allows for seamless bi-directional conversions between the two subscription models on eligible VM instances.

Customers may see combined savings estimated to up to 76% with Azure Hybrid Benefit for Linux and three-year Azure Reserved VM Instances. 
Savings estimates are based on one standard D2s v5 Azure VM with RHEL or SLES subscription in the East US region running at a pay-as-you-go rate vs a reduced rate for a three-year reserved instance plan. 
Based on Azure pricing as of September 2024. Prices subject to change. Actual savings may vary based on location, instance type, or usage.

> [!TIP]
> Try the **[Azure Hybrid Benefit Savings Calculator](https://azure.microsoft.com/pricing/hybrid-benefit/#calculator)** to visualize the cost saving benefits of this feature.

## Defining Pay-As-You-Go (PAYG) and Bring-Your-Own-Subscription (BYOS)

> In Azure, there are two main licensing pricing options: Pay-As-You-Go (PAYG) and Bring-Your-Own-Subscription (BYOS). 

Reviewer: ```suggestion
In Azure, there are two main licensing pricing options: Pay-as-you-go and Bring-Your-Own-Subscription (BYOS). 
```

> With PAYG you only pay for the resources you use and can scale up or down as needed. 

Reviewer: ```suggestion
With pay-as-you-go models you only pay for the resources you use and can scale up or down as needed. 
```

BYOS enables you to use your existing license subscriptions, in this case RHEL and SLES, with no need to pay license fees in Azure.

:::image type="content" source="./media/ahb-linux/azure-hybrid-benefit-compare.png" alt-text="Diagram that shows the use of Azure Hybrid Benefit to switch Linux virtual machines between pay-as-you-go and bring-your-own-subscription.":::

> [!NOTE]
> Virtual machines deployed from PAYG images or VMs converted from BYOS models incur *both* an infrastructure fee and a software fee. If you have your own license, use Azure Hybrid Benefit to convert from a PAYG to BYOS model.

You can use Azure Hybrid Benefit to switch between the two license options at any time.

## Which Linux Virtual Machines are Eligible for Azure Hybrid Benefit 

> ### PAYG 
> The following PAYG RHEL and SLES Marketplace offers are AHB eligible:
> 
> ##### [RHEL PAYG](#tab/ahbRhelPayg)
> ##### RHEL PAYG offers that are ABH eligible:
> **All of the RHEL PAYG eligible offers must have Red Hat, Inc. as the publisher.**

Reviewer: ```suggestion
### Pay-as-you-go 
The following pay-as-you-go RHEL and SLES Marketplace offers are AHB eligible:

##### [RHEL pay-as-you-go](#tab/ahbRhelPayg)
##### RHEL pay-as-you-go offers that are ABH eligible:
**All of the RHEL pay-as-you-go eligible offers must have Red Hat, Inc. as the publisher.**
```

The names of the eligible offers are listed below, with links to them in the Azure Marketplace.
Within these offers, the associated images will be described as 'Pay-As-You-Go'.

* [Red Hat Enterprise Linux ( RHEL )](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/redhat.rhel-20190605?tab=Overview)
* [Red Hat Enterprise Linux ( RHEL ) for SAP Apps](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/redhat.rhel-sap-apps?tab=Overview)
* [RHEL for SAP with HA and Update Services](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/redhat.rhel-sap-ha?tab=Overview)
* [Red Hat Enterprise Linux ( RHEL ) with High Availability ( HA ) Add-On](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/redhat.rhel-ha?tab=Overview)

> * [Red Hat Enterprise Linux ( RHEL ) ARM64](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/redhat.rhel-arm64?tab=Overview)

Reviewer: ```suggestion
* [Red Hat Enterprise Linux ( RHEL ) Arm64](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/redhat.rhel-arm64?tab=Overview)
```

* [Red Hat Enterprise Linux Confidential VM](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/redhat.rhel-cvm?tab=Overview)
* [Red Hat Enterprise Linux ( RHEL ) RAW](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/redhat.rhel-raw?tab=Overview)

> ##### [SLES PAYG](#tab/ahbSlesPayg)
> ##### SLES PAYG offers that are ABH eligible:
> **All of the SLES PAYG eligible offers must have SUSE as the publisher.**

Reviewer: ```suggestion
##### [SLES pay-as-you-go](#tab/ahbSlesPayg)
##### SLES pay-as-you-go offers that are ABH eligible:
**All of the SLES pay-as-you-go eligible offers must have SUSE as the publisher.**
```

The names of the eligible offers are listed below, with links to them in the Azure Marketplace.
Within these offers, the associated plans and images will be described as a 'Pay as You Go' subscription of SLES.
* [SUSE Enterprise Linux 15 SP5 + 24x7 Support](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/suse.sles-15-sp5?tab=Overview)

> * [SUSE Enterprise Linux 15 SP5 arm64 + 24x7 Support](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/suse.sles-15-sp5-arm64?tab=Overview)

Reviewer: ```suggestion
* [SUSE Enterprise Linux 15 SP5 Arm64 + 24x7 Support](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/suse.sles-15-sp5-arm64?tab=Overview)
```
